.github/workflows/sync-issue-with-pr.yaml

```
@@ -0,0 +1,75 @@
+              done
+            fi
+
+            # Copy only major, minor, and patch labels from the issue to the PR
+            for LABEL in $LABELS
+            do
+              if [[ "$LABEL" == "major" || "$LABEL" == "minor" || "$LABEL" == "patch" ]]; then
+                echo "Adding label $LABEL to PR"
+                gh pr edit ${{ github.event.pull_request.number }} --add-label "$LABEL"
+              fi
+            done
+          done
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)